packages/vats/src/core/types.js

```diff
@@ -173,7 +173,15 @@
  *   installation:{
  *     produce: Record<WellKnownName['installation'], Producer<Installation>>,
  *     consume: Record<WellKnownName['installation'], Promise<Installation<unknown>>> & {
+ *       centralSupply: Promise<Installation<import('@agoric/vats/src/centralSupply.js').start>>,
+ *       committee: Promise<Installation<import('@agoric/governance/src/committee.js').start>>,
+ *       contractGovernor: Promise<Installation<import('@agoric/governance/src/contractGovernor.js').start>>,
+ *       econCommitteeCharter: Promise<Installation<import('@agoric/inter-protocol/src/econCommitteeCharter.js').start>>,
+ *       feeDistributor: Promise<Installation<import('@agoric/inter-protocol/src/feeDistributor.js').start>>,
  *       mintHolder: Promise<Installation<import('@agoric/vats/src/mintHolder.js').prepare>>,
+ *       psm: Promise<Installation<import('@agoric/inter-protocol/src/psm/psm.js').start>>,
+ *       reserve: Promise<Installation<import('@agoric/inter-protocol/src/reserve/assetReserve.js').start>>,
+ *       VaultFactory: Promise<Installation<import('@agoric/inter-protocol/src/vaultFactory/vaultFactory.js').start>>,
```

Comment:
This looks like a significant regression. I need to understand the motivation better.

There was some attempt to layer `EconomyBootstrapSpace` on top rather than putting that stuff here.

https://github.com/Agoric/agoric-sdk/blob/7c17a16d3396f1842ec971cba4f4580eb82238d5/packages/inter-protocol/src/proposals/econ-behaviors.js#L86-L87

Reply:
I'll put these wherever they go. I just command-clicked `installation` in the param destructuring to find out where the defs were coming from. I'll try `EconomyBootstrapSpace`

Reply:
Now I remember why I put them here: the [keys are from `WellKnownName`](https://github.com/Agoric/agoric-sdk/blob/7712dcb61770a8b48f133e6c1a1a720b7803d2b6/packages/vats/src/core/types.js#L136-L143) which is the key of the `installation` property Record.

Changing that is out of scope, imo. Acceptable?

Reply:
yes.

Reply:
oic how this works now: `econCommitteeCharter` is not just any old installation... got it.